### DISCUSSION TITLE
Why would a processed food factor be lower than the associated fresh food?​

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/supply-chain-factors#7] [Q&A] [answered]
Using "Poultry" as an example:
- Poultry Farming = 0.77 kg CO2/$
- Packaged Poultry = 0.61 kg CO2/$

**Comment by MoLi7** [comment, marked as answer]
Input of Poultry Farming = $0.43 Poultry Farming per $ Packaged Poultry
Contribution from Poultry Farming to Packaged Poultry CO2 = 0.43 * 0.77 = 0.33 kg CO2/$ Packaged Poultry
Contribution from other commodities + direct CO2 = 0.28 kg CO2/$ Packaged Poultry

Emission factor of Packaged Poultry = 0.33 + 0.28 = 0.61 kg CO2/$

**Reply by skippaah** [reply]
Hello Mo,

I would like to make sure I understand the reasoning for a higher EF for farmed poultry even though it would make sense that there is less carbon emissions associated with farmed poultry vs processed poultry. 

Assumption or Reason Farmed Poultry EF is higher than Processed Poultry:  **Cost of Product influences size of emission factor:** The emission factor for farmed poultry is higher because buying a whole chicken will be cheaper per pound than a processed chicken so the emission factor will be higher to reflect the relationship of cost per pound difference between the 2 different types of chicken.  

Examples showing cost impacts total emissions and shows why processed chicken which is more expensive than a whole chicken has a higher CO2, even though it has a lower EF:
Example 1- 10 lbs whole chicken @ $2.49 per pound = 10 X $2.49 x .77 = 19.173 kg CO2

Example 2- 10 lbs processed chicken thighs @ $5.49 per pound = 10lbs x $5.49 x .61 = 33.489 kg CO2

Example 3- 10lbs processed chicken breasts @ $5.99 = 10lbs x $5.99 x .61 = 36.539 kg CO2

Cost data from Costco
Boneless Skinless Thighs $5.49/lb
Whole $2.49/lb
Wings $3.99/lb
Boneless Skinless Breasts $5.99/lb

Explanation of why Processed Chicken only assumes .43 cents of the $1 for Farmed Chicken: **Processed Chicken is removed early from the whole 'process' of a whole chicken:** The processed chicken only assumes .43 cents on the full dollar of a farmed chicken (the processed chicken is removed early from the whole chicken total process of being sold and therefore has to have some of the carbon footprint removed) and so that will lower the emissions factor before you add in the other associated emission factors for the processed chicken. 

Thanks for the help! 

Cheers,
Keith

**Reply by MoLi7** [reply]
Hi Keith,

Thanks for posting your interpretation and a reasonable example. In general, your understanding is correct.
> The processed chicken only assumes .43 cents on the full dollar of a farmed chicken (the processed chicken is removed early from the whole chicken total process of being sold and therefore has to have some of the carbon footprint removed) and so that will lower the emissions factor before you add in the other associated emission factors for the processed chicken.

Cheers,
Mo

**Reply by skippaah** [reply]
Hi Mo,

Thanks so much for the reply. Can you also please confirm if my understanding is also correct regarding the cost of a product influence on its designated emission factor? 

_Assumption or Reason Farmed Poultry EF is higher than Processed Poultry: Cost of Product influences size of emission factor: The emission factor for farmed poultry is (partly) higher because buying a whole chicken will be cheaper per pound than a processed chicken so the emission factor will be impacted (larger number) to reflect the relationship of cost per pound difference between the 2 different types of chicken._

Thanks for the help!
Keith

**Reply by MoLi7** [reply]
Hi Keith,

Your interpretation _Cost of Product influences size of emission factor_ sounds correct, but it's not the most intuitive way to understand what causes the different EFs between _farmed chicken_ and _processed chicken_.

Essentially, EF of _processed chicken_ depends on 1) how much _farmed chicken_ is attributed as input to _processed chicken_, in this example it's 43% (43 cents of 1 dollar) which yields an EF of 0.33 kg CO2/$ Packaged Poultry, and 2) how large the contribution from other commodities (used for processing _farmed chicken_) is, in this example the associated EF is 0.28 kg CO2/$ Packaged Poultry. Just like how EF of part 1 and 2 together determine the EF of _processed chicken_, cost of part 1 and part 2 together determine the cost of _processed chicken_. Therefore, _cost of product_ is more likely to be a metric related to EF than the cause of EF.

I hope this is helpful,
Mo

**Reply by skippaah** [reply]
Hi Mo,

Thank you so much for your feedback on my question. Your information is super helpful in understanding a complex process. Thank you! 

Keith
